pkg/apis/local/v1/types.go

```diff
@@ -1,14 +1,19 @@
 package v1
 
 import (
+	"os"
+
 	operatorv1 "github.com/openshift/api/operator/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	defaultDiskMakerImageVersion = "registry.svc.ci.openshift.org/ocp/4.2:local-storage-diskmaker"
-	defaultProvisionImage        = "registry.svc.ci.openshift.org/ocp/4.2:sig-storage-local-static-provisioner"
+	defaultDiskMakerImageVersion = "registry.svc.ci.openshift.org/origin/4.2:local-storage-diskmaker"
```

Comment:
IMO,  quay.io/openshift/origin-local-storage-diskmaker looks nicer as default. In theory, they should be the same though.

And I am not sure that baking version (4.2) in code is a good idea, we probably don't want to bump it before every release.

Reply:
Yeah using quay.io/openshift image works. I am just trying to figure out if we can use quay.io/openshift image for sig-storage-local-static-provisioner too otherwise we might have to use upstream image..